cmd/boots/main.go

```diff
@@ -17,16 +30,19 @@ import (
 	"github.com/tinkerbell/boots/metrics"
 	"github.com/tinkerbell/boots/packet"
 	"github.com/tinkerbell/boots/syslog"
-	"github.com/tinkerbell/boots/tftp"
+	"github.com/tinkerbell/ipxedust"
+	"github.com/tinkerbell/ipxedust/ihttp"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"golang.org/x/sync/errgroup"
+	"inet.af/netaddr"
 
```

Comment:
can you remove this empty line here? (hmm boots needs some linter updates)

Reply:
updated